### PR TITLE
Edits to wp-install.rst

### PR DESCRIPTION
**Pull request by deb-intel**
Signed-off-by: deb-intel <38864784+deb-intel@users.noreply.github.com>

### CODE REVIEW

File: source/clear-linux/tutorials/wp-install/wp-install.rst [hunk 1]
```diff
@@ -1,215 +1,170 @@
...
-Click the :guilabel:`Let’s go!` button to enter the information.
+    `Figure 2: WordPress Welcome screen.`
 
-Enter the database name, username, and password we used when creating the
-database:
+#. Click :guilabel:`Let’s go!`.
+#. Enter database connection specifics in the screen that appears as shown in Figure 3 below.
 
-   Database name:       WordPress
+   - Database name:       WordPress
```
Comment (mvincerx) [inline source/clear-linux/tutorials/wp-install/wp-install.rst]: Just FYI: Bullets below are acceptable. 
In contrast, the CL team generally uses '*' for bullets. So going forward, please use the latter style.

File: source/clear-linux/tutorials/wp-install/wp-install.rst [hunk 2]
```diff
@@ -1,215 +1,170 @@
...
-attacks. The security of your website and the data it contains are complex
-and ever-evolving tasks. They should be at the top of your to do list if you
-plan to expose your website to the outside world. This tutorial does not
-address security measures that you can take to harden your site but we
-strongly encourage you to look into it.
+.. note::
 
-Installing WordPress
---------------------
+   Throughout this tutorial, we reference your website name as <your_website>.
```
Comment (mvincerx) [inline source/clear-linux/tutorials/wp-install/wp-install.rst]: Please revise the note as follows: 

.. note::
  [empty line] 
   Throughout this tutorial, ...

Note: Enter a hard return after `.. note:: `. Enter an empty line. Then press `Spacebar` to align the 'T' in Throughout with the 'n' in `note`.

Comment (mvincerx) [reply source/clear-linux/tutorials/wp-install/wp-install.rst]: In this GitHub GUI, the formatting will not appear correctly. For another example, see the first note in `bare-metal-install.rst`

Comment (deb-intel) [reply source/clear-linux/tutorials/wp-install/wp-install.rst]: Hi Michael. Thanks for the tip. I entered this in the file (empty line, spacebar, align) although it does not show up in the diff. Just FYI.

Comment (mvincerx) [reply source/clear-linux/tutorials/wp-install/wp-install.rst]: The extra line and indentation for the body of the note look correct.  Nice work.

File: source/clear-linux/tutorials/wp-install/wp-install.rst [hunk 3]
```diff
@@ -1,215 +1,170 @@
 .. _wp-install:
 
-Create a Clear Linux WordPress\* server
+Create a Clear Linux\* WordPress\* server
 #######################################
 
-Prerequisites
-=============
+This tutorial shows you how to install the WordPress components on your |CL| LAMP server. At the end of :ref:`Creating a Clear Linux based web server <web-server-install>`, you created a
```
Comment (mvincerx) [inline source/clear-linux/tutorials/wp-install/wp-install.rst]: Sorry. It was my error. We need to also add an asterisk to first instance of Linux in the body text too.  
Please add asterisk as follows: `:ref: `Creating a Clear Linux\* based web server.`

Comment (rcaballeromx) [reply source/clear-linux/tutorials/wp-install/wp-install.rst]: Let me clarify some rather important things:
The asterisk should only be added on the first instance in the body of the document of a specific trademarked named owned by an entity other than Intel.
Asterisks in headings should be avoided as much as possible! They create problems with cross-references. @mvincerx @deb-intel You are now seeing one example of said problems.
Additionally, the cross-reference you are adding has a label, `web-server-install`. It is incorrect to add the heading's text here since the \:ref\:\`web-server-install\` role will automatically use it for the link upon rendering.
In short, replace the `ref` role with \:ref\:\`web-server-install\` and everything should render properly.

Comment (mvincerx) [reply source/clear-linux/tutorials/wp-install/wp-install.rst]: IMHO, I believe our team would be better served if the use of asterisks were documented in our Contributor Guidelines. This would be more fair to external and internal contributors.  

As for the :ref: directive, there are good reasons to use "link text" instead of just using a link :ref:`<web-server-install>` to the html page. For example, it would also be correct to write: 

[...] At the end of :ref: `the web server installation guide <web-server-install>`, you created a [...]
This style is widely used in CL documentation. 

Note: Formatting for the above may not render correctly here. 

On the other hand, I see your point on avoiding redundancy in the header when using :ref:.

**Comment by mvincerx** [comment]
@mltullis, @rcaballeromx, This PR is ready for approval and merge when appropriate.

**Comment by mltullis** [comment]
@deb-intel and @mvincerx 
Thanks Deb for this contribution! This is merged, and you can now delete your branch.
